src/client/eppo-client.ts

```diff
@@ -19,8 +17,8 @@ import {
 import { decodeFlag } from '../decoding';
 import { EppoValue } from '../eppo_value';
 import { Evaluator, FlagEvaluation, noneResult } from '../evaluator';
-import ExperimentConfigurationRequestor from '../flag-configuration-requestor';
-import HttpClient from '../http-client';
+import FlagConfigurationRequestor from '../flag-configuration-requestor';
```

Comment:
the default exports in nodejs are kind of funny to me; noticed this wasn't renamed.